code/rdb/rdbstandard.q

```diff
@@ -1,5 +1,27 @@
 // Get the relevant RDB attributes
-.proc.getattributes:{`date`tables!(.rdb.rdbpartition[],();tables[])}
+.proc.getattributes:{default:`date`tables`procname!(.rdb.rdbpartition[];tables[] except .rdb.ignorelist;.proc.procname);
+    / get all cols that contains date (of type "pdz")
+    timecolumns:1!{tcols:exec c from meta value x where t in"pdz";
```

Comment:
```suggestion
    timecolumns:1!{tcols:exec c from meta x where t in"pdz";
```

don't need to value tablename

is the `timecolumns` structure actually needed here? seems like we reference `tcols` later, but I think for `timecolumns` we only really use `raze[value each timecolumns][;0]` - it might be simpler if we didn't build this structure here & just had separate variables for `tcols` and whatever the other bit of `timecolumns` is that we use later on? (just noticed it is joined to something later, so maybe it is needed - it still may be simpler to use some intermediate variables here, get the date range from those etc., then construct the final structure from those later in function)